app/models/early_career_payments/eligibility.rb

```diff
@@ -1,5 +1,128 @@
 module EarlyCareerPayments
   class Eligibility < ApplicationRecord
+    AwardAmount = Struct.new(
+      :itt_subject,
+      :itt_academic_year,
+      :claim_academic_year,
+      :base_amount,
+      :uplift_amount,
+      keyword_init: true
+    )
```

Comment:
@nomasprime How about this being considered as a class or at very least a concern that is included?